bpf/lib/eps.h

```diff
@@ -31,6 +31,8 @@ struct endpoint_key {
 
 #define ENDPOINT_F_HOST			1 /* Special endpoint representing local host */
 #define ENDPOINT_F_ATHOSTNS		2 /* Endpoint located at the host networking namespace */
+#define ENDPOINT_F_NO_SNAT_V4	4 /* Endpoint should not be masqueraded for IPv4 */
+#define ENDPOINT_F_NO_SNAT_V6	8 /* Endpoint should not be masqueraded for IPv6 */
```

Comment:
Unclear whether we need two flags for this on the datapath level.  Are we sharing endpoint map entries for an endpoint, so that the lookup for an endpoint's IPv4 and IPv6 address would return the **same** struct? That would feel rather odd.